pom.xml

```diff
@@ -1747,6 +1747,18 @@
                 </exclusions>
             </dependency>
 
+            <dependency>
+                <groupId>org.apache.iceberg</groupId>
+                <artifactId>iceberg-aws</artifactId>
```

Comment:
I think the only thing being referenced from this project is the `S3FileIOProperties`, so it might make sense to just drop this dependency and use constants for those properties.  The values are part of the REST Spec and shouldn't change.